### DISCUSSION TITLE
Make email more useful

### DISCUSSION THREAD

**Discussion by vogelap** [sct/overseerr#3359] [Feature Requests]
The email received ("Movie Request Now Available") could be made more useful if the movie info was a link back to Overseerr.

Could it be possible that the email could also feature some management tools? Specifically, for me, I would like to be able to UNMONITOR the movie (in Radarr) by pressing a button in the email. Is it possible to add that feature?

**Comment by TheCatLady** [comment]
There is a link if you've configured the `Application URL` setting of your Overseerr instance.

**Comment by vogelap** [comment]
Thanks for that! I didn't know! So, I just put "http://www.(domain).com/5055" in `Application URL` setting?
